scripting/include/srccoop/classdef.inc

```diff
@@ -780,7 +781,7 @@ methodmap CBaseEntity
 	}
 	public void SetCollisionBounds(const float vecMins[3], const float vecMaxs[3])
 	{
-		SDKCall(g_pSetCollisionBounds, this.GetEntIndex(), vecMins, vecMaxs);
+		SDKCall(g_pSetMinMaxSize, this.GetEntIndex(), vecMins, vecMaxs);
```

Comment:
Are there functional differences or side effects between SetCollisionBounds and SetMinMaxSize?

Reply:
it's a wrapper, see https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/mp/src/game/server/util.cpp#L1282